tron/utils/scribereader.py

```diff
@@ -183,6 +186,15 @@ def read_log_stream_for_action_run(
         finally:
             stream.close()
 
+    if truncated_output:
+        formated_today = datetime.datetime.now(datetime.timezone.utc).strftime("%Y-%m-%dT%H:%M:%S.%fZ")
+        output.append(
+            (
+                formated_today,
+                f"This output is truncated. Use this command to view all lines 'scribereader -s {paasta_cluster} -f stream_paasta_app_output_{namespace}_{job_name}__{action}'",
```

Comment:
```suggestion
                f"This output is truncated. Use this command to view all lines 'scribereader -s {paasta_cluster} -f {stream_name}'",
```

Reply:
i *think* we might also need to maybe add `--min-date` and `--max-date` here so that this line is accurate even if someone is looking at a log a week later?